src/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: white;
+  width: 198px;
+  height: 401px;
+}
+
+.product__img {
+  width: 160px;
+  height: 134px;
+  margin: 30px auto;
+}
+
+.product__title {
+  width: 170px;
+  font-size: 12px;
+  line-height: 15px;
+  height: 31px;
+  margin: 15px 14px 0;
+  font-weight: 500;
+}
+
+.product__code {
+  font-weight: normal;
+  margin: 5px 14px;
+}
+
+.product__stars,
+.product__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 15px;
+}
+
+.product__container {
+  display: flex;
+  justify-content: space-between;
+  width: 91px;
+}
+
+.star {
+  display: flex;
+  width: 15px;
+  height: 15px;
+  
+}
+
+.product__cell {
+  background: url(images/star-active.svg) no-repeat center;
+}
+
+.product__cell--last {
+  background: url(images/star.svg) no-repeat center;
+}
```

Comment:
use your code from previous task, so when I change class in HTML - count of active stars also changes

Reply:
see the last commit
I don't understand what the problem in my code(